packages/records/lib/db/migrations/20250823172055_mark_as_deleted_index.ts

```diff
@@ -0,0 +1,23 @@
+import type { Knex } from 'knex';
+
+export const config = {
+    transaction: false
+};
+
+export async function up(knex: Knex): Promise<void> {
+    // dropping existing indexes that are not being used by pg
+    // replacing with a more specific index below used by the "mark previous generation as deleted" query
+    await knex.raw(`DROP INDEX IF EXISTS records_sync_id_index;`);
+    await knex.raw(`DROP INDEX IF EXISTS records_sync_job_id_index;`);
```

Comment:
Can we just drop without iterating over each table?

Reply:
the DROPs isn't iterating but removing the parent indexes.